filebeat/harvester/reader/line.go

```diff
@@ -121,6 +129,9 @@ func (l *Line) advance() error {
 	// found encoded byte sequence for '\n' in buffer
 	// -> decode input sequence into outBuffer
 	sz, err := l.decode(idx + len(l.nl))
+	if err != nil {
+		logp.Err("Error decoding line: %s", err)
+	}
```

Comment:
decode method should reset `err=nil` on `transform.ErrShortDst`. `ErrShortDst` is transformer complaining about output buffer being full+more bytes in input buffer available for transcoding.